packages/platform-sdk-profiles/src/drivers/memory/wallets/wallet.ts

```diff
@@ -54,8 +54,7 @@ export class Wallet implements IReadWriteWallet {
 	readonly #transactionIndex: ITransactionIndex;
 	readonly #walletImportFormat: IWalletImportFormat;
 	readonly #multiSignature: IMultiSignature;
-	readonly #decimals = 8; // TODO: use value from manifest
-	readonly #denomination = BigNumber.powerOfTen(this.#decimals);
+	readonly #decimals = 8; // TODO: use value from manifest?
```

Comment:
As the comment says, we should grab this value from the network manifest which should be accessible from within this instance.

Reply:
Am I supposed to pass the config all the way down to the `WalletFactory`?

Reply:
Not sure why you would need to pass it to the wallet factory? You should be able to do `this.#decimals = this.manifest().get<number>("network.currency.decimals");` at some point in the lifecycle. I'll take a look tomorrow morning if it's not sorted by then.

Reply:
I have this error when I try to call `this.manifest()` in `Wallet#constructor` 

![image](https://user-images.githubusercontent.com/1091900/120676919-d1dd1e80-c496-11eb-8e83-4f9ce642f5c1.png)

Reply:
The manifests only is accessible after a coin has been set.